integration/healthcheck_test.go

```diff
@@ -92,3 +92,89 @@ func (s *HealthCheckSuite) TestSimpleConfiguration(c *check.C) {
 	c.Assert(err, checker.IsNil)
 	c.Assert(resp.StatusCode, checker.Equals, http.StatusNotFound)
 }
+
+func (s *HealthCheckSuite) TestMultipleEntrypoints(c *check.C) {
+	whoami1Host := s.composeProject.Container(c, "whoami1").NetworkSettings.IPAddress
```

Comment:
Could you refactor tests like that:

```golang
type HealthCheckSuite struct{
	BaseSuite
	whoami1IP string
	whoami2IP string
}

func (s *HealthCheckSuite) SetUpSuite(c *check.C) {
	s.createComposeProject(c, "healthcheck")
	s.composeProject.Start(c)

	s.whoami1IP = s.composeProject.Container(c, "whoami1").NetworkSettings.IPAddress
	s.whoami2IP = s.composeProject.Container(c, "whoami2").NetworkSettings.IPAddress
}
```

and replace all occurrence of `whoami1Host` and `whoami2Host` by `s.whoami1IP` and `s.whoami2IP`